### PR TITLE
Introduce GitHub Repo Configurator

### PR DESCRIPTION
**Pull request by tomkerkhove**
Introduce GitHub Repo Configurator which will help us onboard new projects without any impact.

- As of now, @arcus-automation should be added to all our repos by default.
- A more fine-grained list of all labels, issues & milestones will be added in follow-up PR

Relates to #2 

## Running the tool
Run the tool
```shell
Arcus.Tools.Repository.App.exe
        --username "<github-username>"
        --password "<github-password>"
        --repo-name "<github-repo>" #  example: arcus/arcus.keyvault
        --configuration-file "<path-to-configuration>"
```

## Manifest
A manifest is used to describe what labels, issues and milestones are required.

```yaml
labels:
  - name: management
    description: All issues related to management of the project
    color: f282af
issues:
  - title: Provide NuGet badge
    description: Provide NuGet badge in the README
    milestoneName: Project Setup
    labels:
    - management
milestones:
 - title: Project Setup
   description: All tasks related to creating a new Arcus project
```

### CODE REVIEW

**Comment by tomkerkhove** [comment]
I've written better things but this should be good enough for an automation tool imo.

File: tools/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Arcus - Tools
+Arcus provides some tooling to streamline the governance & setup of new projects.
+
+- **GitHub Repo Configurator** - Configures a GitHub repository according to a defined repository manifest.
```
Comment (samneirinck) [inline tools/README.md]: Would make this a link to the README.md of that project.

File: tools/github-repo-configurator/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Building the tool
+You can easily build the tool with the .NET Core commands:
+```shell
+dotnet build .\src
+```
+
+## Running the tool
+Run the tool
+```shell
+Arcus.Tools.Repository.App.exe
```
Comment (samneirinck) [inline tools/github-repo-configurator/README.md]: Does `dotnet build` produce an `.exe`? I would've thought it was a `.dll`, if it was a .NET Core project.

File: tools/README.md (hunk 1 above)
Comment (tomkerkhove) [reply tools/README.md]: Good suggestion!

File: tools/github-repo-configurator/README.md (hunk 2 above)
Comment (tomkerkhove) [reply tools/github-repo-configurator/README.md]: Good point, I actually need to check this - Assumed that this was the case which is wrong.

**Comment by tomkerkhove** [comment]
Should be good to go now @samneirinck. Let me know what you think as well @SamVanhoutte

**Comment by tomkerkhove** [comment]
Example of running the tool with default config:
image

Created milestone:
image

Created issues:
image

Created labels:
image

Have a look here on [tomkerkhove/github-automation](https://github.com/tomkerkhove/github-automation) - This looks good for a first version.

**Pull request merged**